index.js

```diff
@@ -1,5 +1,7 @@
 /*! standard. MIT License. Feross Aboukhadijeh <https://feross.org/opensource> */
-const Linter = require('standard-engine').linter
-const opts = require('./options')
+import engine from 'standard-engine'
+import opts from './options.js'
```

Comment:
Not related to this PR, but I think it would be nice to rename `opts` to `options` ☺️